db/citations.yml

```diff
@@ -2289,13 +2289,16 @@ bozer;aca22:
   grant: 'dmref19'
   journal: 'Acta Crystallogr. A'
   nb: ''
-  note: ''
+  note: 'arXiv:2204.00434v1 [cond-mat.mtrl-sci]'
   number: ''
   optannote: ''
   optwwwlanl: ''
   pages: ''
   title: 'Towards a machine-readable literature:
           finding relevant papers based on an uploaded powder diffraction pattern'
+  tags: 'first_submission_IUCr'
+  synopsis: ''
+  professional_summary: ''
```

Comment:
this is longer.  it needs a bit of context.  Something that is a mixture of the paper introduction and the abstract, but only as long as the abstract.  Can you have a crack at it?  You can look in the db for examples.  The target audience is scientists who might be interested (the target audience of our use-case, so someone like Sandy or Dave Billing if he didn't know about the project).

We should also add a public_summary which would be written for the general public.  No jargon, no details.  Have a crack at that too, why not.  This would be something that your dad or mum could understand.....

Reply:
Done. 
@sbillinge : Both public and professional summaries ready for review.